images/fcos-bastion-image/root/usr/lib/systemd/system/dhcp-init.service

```diff
@@ -3,7 +3,6 @@ Description=Init configuration for dhcp and tftpboot with dnsmasq
 RequiresMountsFor=%t/containers
 Wants=network-online.target
 After=network-online.target
```

Comment:
can you confirm this is safe to do with the current implementation of the initializer script?

Reply:
Removing `ConditionPathExists` would regenerate couple of ignition files, extract grub efi and copy original conf (read-only) from the local path removing any modifications done to the conf file in use. Do we want that on the running bastion host?

If the question is about the `Wants` and `After` systemd directives then I think it should not have any issue.

Reply:
You're right.

What I'm asking is whether we can do it today, if we can verify all is good for this to happen. I would prefer that as that will ensure that this repo's configuration is well reflected at runtime and will drop any manual patch. 

However, initially, we needed to change the DHCP conf in particular because we weren't using the folder-based approach in the ephemeral clusters, so that wasn't possible.

Today, unless there is some other specific need and as far as the currently deployed conf is exactly the same as the one that we would 'regenerate' at each boot, I would prefer forcing the conf computed by this unit.

Reply:
Oh, it is not ready to do it today. I will add one commit to remove `router` option for IPv6 (dnsmasq fails to start). I  need to figure out why the IPv6 prefix-length is set to 128 instead of 64 causing IPv6 connectivity issue. This will be tough to test with this change.

Reply:
The reset would only happen at reboot.

Reply:
Also, double check that need for a /64 prefix. I think /128 (+ RA) is good for hosts, but I'm not 100% sure